examples/unet_segmentation_3d_dict.py

```diff
@@ -28,42 +28,52 @@
 
 import monai
 import monai.transforms.compose as transforms
-from monai.data.nifti_reader import NiftiDatasetd
-from monai.transforms.composables import AddChanneld, RandRotate90d
+from monai.transforms.composables import \
+    LoadNiftid, AsChannelFirstd, RandCropByPosNegLabeld, RandRotate90d
 from monai.handlers.stats_handler import StatsHandler
 from monai.handlers.mean_dice import MeanDice
 from monai.visualize import img2tensorboard
 from monai.data.synthetic import create_test_image_3d
 from monai.handlers.utils import stopping_fn_from_metric
+from monai.data.utils import list_data_collate
 
 monai.config.print_config()
 
 # Create a temporary directory and 50 random image, mask paris
 tempdir = tempfile.mkdtemp()
 
 for i in range(50):
-    im, seg = create_test_image_3d(128, 128, 128)
+    im, seg = create_test_image_3d(128, 128, 128, channel_dim=-1)
 
     n = nib.Nifti1Image(im, np.eye(4))
-    nib.save(n, os.path.join(tempdir, 'im%i.nii.gz' % i))
+    nib.save(n, os.path.join(tempdir, 'img%i.nii.gz' % i))
 
     n = nib.Nifti1Image(seg, np.eye(4))
     nib.save(n, os.path.join(tempdir, 'seg%i.nii.gz' % i))
 
-images = sorted(glob(os.path.join(tempdir, 'im*.nii.gz')))
+images = sorted(glob(os.path.join(tempdir, 'img*.nii.gz')))
 segs = sorted(glob(os.path.join(tempdir, 'seg*.nii.gz')))
+train_files = [{'img': img, 'seg': seg} for img, seg in zip(images[:40], segs[:40])]
```

Comment:
It's safer to set a variable and use that instead of 'img' and 'seg' everywhere.

Reply:
Wenqi suggested to unify all constant values after GTC.
Including image properties, networks, etc.
Thanks.